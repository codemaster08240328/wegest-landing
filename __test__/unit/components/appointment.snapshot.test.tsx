import renderer from 'react-test-renderer'
import Appointment from '@/components/appointment'

describe('<Appointment> snapshot', () => {
  it('renders correctly without points props', () => {
    expect(renderer.create(<Appointment />).toJSON()).toMatchSnapshot()
  })

  it('renders correctly with points props', () => {
    expect(
      renderer.create(<Appointment points={[3, 2, 4]} />).toJSON()
    ).toMatchSnapshot()
  })
})
