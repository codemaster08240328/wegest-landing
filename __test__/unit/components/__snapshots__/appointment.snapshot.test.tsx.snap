// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Appointment> snapshot renders correctly with points props 1`] = `
<div
  className="MuiContainer-root  makeStyles-root-1 MuiContainer-maxWidthLg"
>
  <h2
    className="makeStyles-title-2"
    data-testid="appt-title"
  >
    YOUR APPOINTMENTS
  </h2>
  <div
    className="MuiGrid-root makeStyles-content-3 MuiGrid-container"
    data-testid="appt-content"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
    >
      <h3
        className="h3-root-5 h3-root-13"
      >
        Today
      </h3>
      <p
        className="p-root-7 p-root-14 makeStyles-todayPoint-4"
      >
        3
      </p>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
    >
      <h3
        className="h3-root-5 h3-root-15"
      >
        Tomorrow
      </h3>
      <p
        className="p-root-7 p-root-16"
      >
        2
      </p>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
    >
      <h3
        className="h3-root-5 h3-root-17"
      >
        2 days later
      </h3>
      <p
        className="p-root-7 p-root-18"
      >
        4
      </p>
    </div>
  </div>
</div>
`;

exports[`<Appointment> snapshot renders correctly without points props 1`] = `
<div
  className="MuiContainer-root  makeStyles-root-1 MuiContainer-maxWidthLg"
>
  <h2
    className="makeStyles-title-2"
    data-testid="appt-title"
  >
    YOUR APPOINTMENTS
  </h2>
  <div
    className="MuiGrid-root makeStyles-content-3 MuiGrid-container"
    data-testid="appt-content"
  >
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
    >
      <h3
        className="h3-root-5 h3-root-6"
      >
        Today
      </h3>
      <p
        className="p-root-7 p-root-8 makeStyles-todayPoint-4"
      >
        0
      </p>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
    >
      <h3
        className="h3-root-5 h3-root-9"
      >
        Tomorrow
      </h3>
      <p
        className="p-root-7 p-root-10"
      >
        0
      </p>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4"
    >
      <h3
        className="h3-root-5 h3-root-11"
      >
        2 days later
      </h3>
      <p
        className="p-root-7 p-root-12"
      >
        0
      </p>
    </div>
  </div>
</div>
`;
